test/cppsuite/test_harness/workload/database_operation.cpp

```diff
@@ -53,7 +54,9 @@ populate_worker(thread_context *tc)
         uint64_t j = 0;
         while (j < tc->key_count) {
             tc->transaction.begin();
-            if (tc->insert(cursor, coll.id, j)) {
+            std::string value =
+              random_generator::instance().generate_pseudo_random_string(tc->value_size);
+            if (tc->insert(cursor, coll.id, tc->key_to_string(j), value)) {
```

Comment:
Can we also buffer the keys here? I am looking in the case where the configuration defines a set key size, then the keys should buffer the remaining numbers with 0s e.g. key size 5 should be 00000, rather than 0

Reply:
The number of 0s varies as `j` changes. I am not sure how we could have a buffer for the key without compromising the readability of the code.